benchmark/cuchembm/datasets/bioactivity.py

```diff
@@ -3,11 +3,12 @@
 import pandas as pd
 import dask.dataframe as dd
 
-from cuchemcommon.fingerprint import calc_morgan_fingerprints
+from cuchembm.metrics.utils import calculate_morgan_fingerprint
```

Comment:
cuchem?

Reply:
Not really. I believe, this is my lack of creative. I was about to name it cuchembenchmark and decided to us bm for benchmark.

Reply:
Can I suggest `cuchembench` or `modelbench`? `bm` could be confused with a lot of other things.